src/text-overflow/TextOverflow.d.ts

```diff
@@ -1,13 +1,18 @@
 import { CSSProperties, PureComponent } from 'react';
+import { Type } from 'tcomb';
 
 export type TextOverflowProps = {
   children?: (self: JSX.Element, isOpen: boolean) => any,
   label: string,
+  popover?: object, // TODO(typo): Omit?
```

Comment:
@ascariandrea as I told you in the other PRs, this should reflect exactly the `@props` defined in this repo.

Please don't copy-paste from alinity-pro, those are the props of the custom wrappers.
If you start from those you will always make mistakes as they are a different thing :)